modules/nf-core/checkm/qa/main.nf

```diff
@@ -33,7 +33,6 @@ process CHECKM_QA {
         $marker_file \\
         $analysis_dir \\
         $coverage \\
-        $exclude \\
```

Comment:
Why did you remove `$exclude` despite defining it above?

Reply:
Oops, this was not intentional